hack/make-rules/update/generated.sh

```diff
@@ -33,9 +33,8 @@ cd "${REPO_ROOT}"
 
 # run the generators
 # TODO: -o "${REPO_ROOT}/../.." is a weird work-around ...
```

Comment:
The work-around is no longer needed?

Reply:
This is from kind (we forked this), and it didn't work without me removing this. I'm not sure what's up there, but when I removed that it generated the code correctly.

Reply:
Maybe just get rid of the todo as well, cause now it just doesn't make sense (the thing it references is gone)